### PR TITLE
Archive directories before sending

### PR DESCRIPTION
**Pull request by progbeat**

### CODE REVIEW

File: creepy/query/__init__.py [hunk 1]
```diff
@@ -151,16 +160,29 @@ def download(self, obj: ProxyObject):
         return self._post(DownloadQuery(obj._id))
 
     def _send_file(self, src_path: str, dst_path: str, exist_ok=False):
-        CHUNK_SIZE = 2**20
+        CHUNK_SIZE = 2**24
```
Comment (romawhite47) [inline creepy/query/__init__.py:163]: Why do you change this value so often? :)

File: creepy/query/__init__.py [hunk 2]
```diff
@@ -151,16 +160,29 @@ def download(self, obj: ProxyObject):
...
+            raise OSError(f"Path exists: '{dst_path}'")
         with self.globals.open(dst_path, 'wb') as dst_f:
             with open(src_path, 'rb') as src_f:
                 while True:
                     chunk = src_f.read(CHUNK_SIZE)
                     if not chunk:
                         break
-                    dst_f.write(chunk)
+                    n = CHUNK_SIZE
+                    for i in range(-3, 1):
```
Comment (romawhite47) [inline creepy/query/__init__.py:173]: Maybe `reversed(range(3))` is better.

File: creepy/query/__init__.py (hunk 1 above)
Comment (progbeat) [reply creepy/query/__init__.py:163]: Are you sure? It's second time I change it.

Comment (romawhite47) [reply creepy/query/__init__.py:163]: I definitely remember that you changed 2^20 -> 2^24 in some previous commit. AFAIK, it was not the first time.

File: creepy/query/__init__.py (hunk 2 above)
Comment (romawhite47) [reply creepy/query/__init__.py:173]: Or even `reversed(range(num_tries))`

File: creepy/query/__init__.py (hunk 1 above)
Comment (progbeat) [reply creepy/query/__init__.py:163]: First time I changed 24->20, this time I changed 20->24, that's it)

**Review by romawhite47 — APPROVED** [review]
LGTM

**Pull request merged**